### PR TITLE
Update Modal.js

### PR DESCRIPTION
**Pull request by stereodenis**

### CODE REVIEW

**Comment by nihgwu** [comment]
IMO it's not necessary

**Comment by stereodenis** [comment]
@nihgwu it's better to teach users use new simple syntax!

**Comment by nihgwu** [comment]
actually that is in ES7 style, I think ES6 style is OK

**Comment by facebook-github-bot-3** [comment]
By analyzing the blame information on this pull request, we identified @ericnakagawa and @mmmulani to be potential reviewers.

File: Libraries/Modal/Modal.js [hunk 1]
```diff
@@ -35,9 +35,8 @@ const RCTModalHostView = requireNativeComponent('RCTModalHostView', null);
  *
  * class ModalExample extends Component {
  *
- *   constructor(props) {
- *     super(props);
- *     this.state = {modalVisible: false};
+ *   state = {
+ *    modalVisible: false
```
Comment (mmmulani) [inline Libraries/Modal/Modal.js]: nit, trailing comma here

**Comment by facebook-github-bot-4** [comment]
@stereodenis updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9825/files/94d660861b3697449b5283d24d1ebc7e99fb0755..8e9c073826899d5fc322c0e8bd42e1f474cb4b91)

**Comment by mmmulani** [comment]
@facebook-github-bot shipit

File: Libraries/Modal/Modal.js [hunk 2]
```diff
@@ -35,9 +35,8 @@ const RCTModalHostView = requireNativeComponent('RCTModalHostView', null);
  *
  * class ModalExample extends Component {
  *
- *   constructor(props) {
- *     super(props);
- *     this.state = {modalVisible: false};
+ *   state = {
+ *    modalVisible: false,
```
Comment (satya164) [inline Libraries/Modal/Modal.js]: Please fix the indentation here

**Comment by facebook-github-bot-0** [comment]
@stereodenis updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9825/files/8e9c073826899d5fc322c0e8bd42e1f474cb4b91..d70e72ed3237ebe5eb71edcb8e4e5c1c6fe778c1)

File: Libraries/Modal/Modal.js (hunk 2 above)
Comment (stereodenis) [inline Libraries/Modal/Modal.js]: @satya164 done!

**Comment by facebook-github-bot-1** [comment]
@stereodenis updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9825/files/8e9c073826899d5fc322c0e8bd42e1f474cb4b91..d70e72ed3237ebe5eb71edcb8e4e5c1c6fe778c1)

**Comment by facebook-github-bot-1** [comment]
@stereodenis updated the pull request - [view changes](https://github.com/facebook/react-native/pull/9825/files/d70e72ed3237ebe5eb71edcb8e4e5c1c6fe778c1..99449ec98c2e8dbc9e3a8dd71f770adca6cc3a51)

**Comment by mmmulani** [comment]
@facebook-github-bot shipit

**Pull request closed without merging**